frontend/src/shared/video-card.tsx

```diff
@@ -18,21 +18,42 @@ export default function VideoCard({ video, progress }: Props) {
   });
 
   const cardBodyClassname = clsx({
-    "card-body justify-end": video.processing,
-    "card-body": !video.processing,
+    "card-body p-0 justify-end": video.processing,
+    "card-body p-0": !video.processing,
```

Comment:
```suggestion
    "card-body justify-end": video.processing,
    "card-body": !video.processing,
```
We don't need this anymore since we're applying in-line styles